src/main/java/mate/jdbc/dao/impl/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,112 @@
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        String request = "SELECT * FROM manufacturers WHERE id = ? AND isDeleted = false;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getManufacturerStatement = connection.prepareStatement(request)) {
+            getManufacturerStatement.setLong(1, id);
+            ResultSet resultSet = getManufacturerStatement.executeQuery();
+            return resultSet.next() ? Optional.of(parseManufacturer(resultSet)) : Optional.empty();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get manufacture with id = " + id, e);
+        }
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        String request = "SELECT * FROM manufacturers WHERE isDeleted = false;";
+        List<Manufacturer> allManufacturers = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getAllManufacturersStatement
+                        = connection.prepareStatement(request)) {
+            ResultSet resultSet = getAllManufacturersStatement.executeQuery();
+            while (resultSet.next()) {
+                allManufacturers.add(parseManufacturer(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get all manufactures from db", e);
+        }
+        return allManufacturers;
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String request = "UPDATE manufacturers "
+                + "SET name = ?, country = ? "
+                + "WHERE id = ? AND isDeleted = FALSE;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement updateManufacturersStatement
+                        = connection.prepareStatement(request)) {
+            updateManufacturersStatement.setString(1, manufacturer.getName());
+            updateManufacturersStatement.setString(2, manufacturer.getCountry());
+            updateManufacturersStatement.setLong(3, manufacturer.getId());
+            return updateManufacturersStatement.executeUpdate() != 0
+                    ? manufacturer : null;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't update manufacturer in db. Id : "
+                    + manufacturer.getId(), e);
+        }
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String request = "UPDATE manufacturers "
+                + "SET isDeleted = TRUE "
+                + "WHERE id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement deleteManufacturersStatement
+                        = connection.prepareStatement(request)) {
+            deleteManufacturersStatement.setLong(1, id);
+            return deleteManufacturersStatement.executeUpdate() != 0;
```

Comment:
This is also a point from common mistakes so please pay attention to it
```suggestion
            return deleteManufacturersStatement.executeUpdate() > 0;
```